using_images/s2i_images/dot_net_core.adoc

```diff
@@ -33,31 +33,40 @@ and Python from within {product-title}.
 [[dot-net-core-supported-versions]]
 == Supported Versions
 
+* .NET Core version 2.0
```

Comment:
Can we order this lowest to highest? (or highest to lowest, but not lowest in the middle)